app/routes.py

```diff
@@ -1,2 +1,342 @@
+    or "total_inventory" not in request_body:
+        return ({
+            "errors": ["Must input required data"]
+        }, 400)
+    else:
+        new_video = Video.from_json(Video, request_body)
+
+        db.session.add(new_video)
+        db.session.commit()
+
+        return {
+            "id": new_video.video_id
+        }, 201
+
+@videos_bp.route("/<video_id>", methods=["GET"])
+def get_video(video_id):
+    '''
+    Gets video by video_id
+    '''
+    video = Video.query.get(video_id)
+
+    if not valid_item(video):
+        return make_response("No video found", 404)
+
+    return video.to_json()
+
+@videos_bp.route("/<video_id>", methods=["PUT"])
+def update_video(video_id):
+    '''
+    Updates specific video information
+    '''
+    video = Video.query.get(video_id)
+    if not valid_item(video):
+        return make_response("No video found", 404)
+
+    form_data = request.get_json()
+
+    video.title = form_data["title"]
+    video.release_date = form_data["release_date"]
+    video.total_inventory = form_data["total_inventory"]
+
+    db.session.commit()
+
+    return {
+        "id": video.video_id,
+        "title": video.title,
+        "release_date": video.release_date,
+        "total_inventory": video.total_inventory,
+        "available_inventory": video.available_inventory
+    }
+
+@videos_bp.route("/<video_id>", methods=["DELETE"])
+def delete_video(video_id):
+    '''
+    Deletes video by video_id
+    '''
+    video = Video.query.get(video_id)
+    if not valid_item(video):
+        return make_response("No video found", 404)
+
+    db.session.delete(video)
+    db.session.commit()
+
+    return {
+        "id": video.video_id
+    }
+
+@rentals_bp.route("/check-out", methods=["POST"])
+def check_out():
+    '''
+    INPUT: JSON with customer_id and video_id
+    OUTPUT: JSON with new rental information pulled from customer,
+    video, and rental table
+
+    Increases customer's videos_checked_out by 1
+    Decreases video's available_inventory
+    Creates due date 7 days from today
+    '''
+    request_body = request.get_json()
+
+    if not valid_data_type(request_body["customer_id"], int) \
+    or not valid_data_type(request_body["video_id"], int):
+        return jsonify("IDs must be an integer"), 400
+
+    customer_id = request_body["customer_id"]
+    video_id = request_body["video_id"]
+
+    customer = Customer.query.get(customer_id)
+    video = Video.query.get(video_id)
+
+    if not valid_item(customer) or not valid_item(video):
+        return make_response("Customer or video not found", 404)
+
+    new_rental = Rental(customer_id=customer_id, video_id=video_id)
+
+    if video.available_inventory == 0:
+        return {
+            "errors": ["No videos available"]
+        }, 400
+    else:
+        customer.videos_checked_out_count += 1
+        video.available_inventory -= 1
+
+    db.session.add(new_rental)
+    db.session.commit()
```

Comment:
I suggest making the whole  business logic of checking out a video in the models.